app/v2/controllers/V2MovementsController.scala

```diff
@@ -330,10 +330,8 @@ class V2MovementsControllerImpl @Inject() (
     }
   }
 
-  def attachLargeMessage(movementId: MovementId): Action[Source[ByteString, _]] = authActionNewEnrolmentOnly.async(streamFromMemory) {
+  def attachLargeMessage(movementId: MovementId): Action[JsValue] = authActionNewEnrolmentOnly.async(parse.json) {
     implicit request =>
```

Comment:
Is this actually attaching anything?

Reply:
Not yet, we will be doing that in the upcoming sprints